Backend/app/rider-platform/rider-app-drainer/package.yaml

```diff
@@ -146,6 +146,7 @@ executables:
     ghc-options:
       - -threaded
       - -rtsopts
+      - -fwhole-archive-hs-libs
```

Comment:
these flags should be there only for local

Reply:
yes, removing redudant .cabal and .yaml changes